packages/next/src/lib/metadata/resolve-metadata.ts

```diff
@@ -247,7 +247,13 @@ export async function collectMetadata(
 export async function accumulateMetadata(
   metadataItems: MetadataItems
 ): Promise<ResolvedMetadata> {
+  const deepClone: <T>(v: T) => T =
+    process.env.NODE_ENV === 'development'
+      ? require('next/dist/compiled/@edge-runtime/primitives/structured-clone')
+          .structuredClone
+      : (v) => v
```

Comment:
why define this in the function body. In prod hopefully the identity function would get DCE'd but if it didn't you're creating a new function every invocation.